index.html

```diff
@@ -55,5 +55,176 @@
+          <div class="card-body">
+            <h2 class="card-title text-white">Profesional Art Printing Data</h2>
+            <p class="card-description text-white">
+              A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <ul class="card-tags child-tags">
+              <li class="card-tag child-tag">
+                HTML
+              </li>
+              <li class="card-tag child-tag">
+                Bootstrap
+              </li>
+              <li class="card-tag child-tag">
+                Ruby
+              </li>
+            </ul>
+          <button type="button" class="card-button child-button">See project</button>
+          </div>
+        </div><div class="card bg-image3 child-card">
+          <div class="card-body">
+            <h2 class="card-title text-white">Profesional Art Printing Data</h2>
+            <p class="card-description text-white">
+              A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
+            <ul class="card-tags child-tags">
+              <li class="card-tag child-tag">
+                HTML
+              </li>
+              <li class="card-tag child-tag">
+                Bootstrap
+              </li>
+              <li class="card-tag child-tag">
+                Ruby
+              </li>
+            </ul>
+          <button type="button" class="card-button child-button">See project</button>
+          </div>
+        </div>
+      </div>      
+    </section>
+    <section class="bg-primary bg-image2 about-section">
+      <div>
+        <p class="text-primary about-text">
+          About me
+        </p>
+        <p class="text-description">
+          Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>        
+        <button type="button" class="card-button btn-about">Get my resume</button>
+        <!-- <img src="images/divider.svg" alt=""> -->
+
+        <hr class="main-line-white">
+        <ul class="no-decoration text-white tech-list">
+          <li class="tech-list">
+            <span>
+              <img class="list-type-img" src="images/AboutMe/Rectangle.svg" alt=""> Language
+            </span>            
+            <ul class="no-decoration dropdown-list">
+              <li>
+                JavaScript
+              </li>
+              <li>
+                Ruby
+              </li>
+              <li>
+                HTML
+              </li>
+              <li>
+                CSS
+              </li>
```

Comment:
- [ ] **Good job on using `ul` tag to group, and `li` for the icons 👏 👏** However it would be better for each language to add **a link (`a` tag)**.
    
    Apart from being semantically correct, the `a` tag will let you connect your language, using `href`. 😎😎